src/moepkg/exmode.nim

```diff
@@ -1091,6 +1098,26 @@ proc shellCommand(status: var EditorStatus, shellCommand: string) =
 
   status.changeMode(currentBufStatus.prevMode)
 
+proc backgroundCommand(status: var EditorStatus) =
+  saveCurrentTerminalModes()
+  exitUi()
+  discard execShellCmd("printf \"Press Enter\"")
+  discard execShellCmd("read _")
+  restoreTerminalModes()
+  status.commandLine.clear
+  status.changeMode(currentBufStatus.prevMode)
+
+proc manualCommand(status: var EditorStatus, manualInvocationCommand: string) =
+  saveCurrentTerminalModes()
+  exitUi()
+
+  # TODO:  Configure a default manual page to show on `:man`.
+  discard execShellCmd(manualInvocationCommand)
```

Comment:
@fox0430

Here, the default manual page can be configured.  If the `manualInvocationCommand` equals `man`, the default page could be shown.  Else, there are arguments to be passed to `man`.